spot_rl_experiments/spot_rl/real_policy.py

```diff
@@ -3,6 +3,8 @@
 # LICENSE file in the root directory of this source tree.
 
 
+import math
```

Comment:
Seems like math is not getting used in this file. 
Can we remove it in that case?

Reply:
removed math